web-sdk/README.md

```diff
@@ -53,6 +53,8 @@ fileName:index.html
 ```
 To integrate **Fidel Web SDK** in your website or mobile app, you need to add the script above in your website or mobile web view. For mobile apps, however, we recommend using the [mobile SDKs](/mobile-sdk).
 
+>Hint: Adding user data in the metadata as key:value pairs can reconcile the cardId to a user account in your system.
```

Comment:
not the cardId, but the metadata id, e.g. if I pass `myUserId: '123'` in metadata, I can match it to user `123` in my system, and [list cards by that id](https://reference.fidel.uk/reference#list-cards-from-metadata-id)